### PR TITLE
Update readme for 2.5.1

### PR DESCRIPTION
**Pull request by Shelob9**
DO NOT MERGE YET.

Blockers to merge:

- [x] Correct release date.
- [x] Ensure #2560 got fixed
- [x] Any last minute changes for changelog

### CODE REVIEW

**Comment by Shelob9** [comment]
Needs #2620

**Comment by Shelob9** [comment]
Is now ready to merge, pending Travis and Scott.

Reassigning to @sc0ttkclark to review and merge.

File: readme.txt [hunk 1]
```diff
@@ -154,6 +154,18 @@ Many thanks go out to the fine folks who have helped us translate Pods into othe
  Join us in further translating the Pods interface at: http://wp-translate.org/projects/pods
 
 == Changelog ==
+= 2.5.1 - January, 22 2015 =
```
Comment (sc0ttkclark) [inline readme.txt]: `= 2.5.1 - January 22nd, 2015 =`

and 2.5 should be:

`= 2.5 - December 30th, 2014 =`

**Comment by sc0ttkclark** [comment]
Merge pending text tweak

**Pull request merged**